files/watch-element-resize.js/update.json

```diff
@@ -0,0 +1,10 @@
+
+{
+  "packageManager": "github",
+  "name": "watch-element-resize.js",
+  "repo": "jonataswalker/watch-element-resize.js",
+  "files": {
+    "basePath": "build/",
+    "include": ["watch-element-resize.min.js"]
```

Comment:
There is no such file here https://github.com/jonataswalker/watch-element-resize.js/tree/1.0.0/build